src/Components/PhoneCertifyBox/PhoneCertifyBox.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import styled from 'styled-components';
+import { theme } from '../../Styles/CommonStyle';
+
+const PhoneCertifyBox = () => {
+  return (
+    <>
+      <h3>휴대폰 번호</h3>
+      <Phone>
+        <div className="inputBox">
+          <input type="number" placeholder={`휴대폰 번호 입력('-')`}></input>
+          <button>인증 요청</button>
+        </div>
+        <input className="inputBox2" type="number" placeholder={`인증번호 입력`}></input>
```

Comment:
윗 줄의 input 요소도 아래와 같이 수정해주세요!
```suggestion
        <input className="inputBox2" type="number" placeholder="인증번호 입력" />
```

Reply:
리뷰를 너무 늦게봤습니다.
이부분은 최신화 되어 없어졌습니다.
확인 감사합니다!